keyboards/dasky/reverb/graphics/display.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2023 Dasky (@daskygit)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "qp.h"
+
+painter_device_t       reverb_display;
+painter_font_handle_t  roboto_font;
```

Comment:
On further inspection, wouldn't this need a `extern`?
Otherwise a `#include` would try and create a second instance of the variables, instead of using the one on `display.c`

Reply:
I think the use of extern is better practice from reading. Switched over in d1f06cc1a9a63a0f4986c6f1fd5c078bd6651ad0